addons/hr_holidays/views/hr_leave_views.xml

```diff
@@ -350,9 +350,7 @@
                 </div>
             </sheet>
             <div class="o_attachment_preview"/>
-            <div class="oe_chatter">
-                <field name="message_follower_ids"/>
-                <field name="message_ids" options="{'post_refresh': 'always'}"/>
+            <div class="oe_chatter" options="{'reload_on_post': True, 'reload_on_attachment': True}">
```

Comment:
We could improve even further:
```xml
<chatter reload_on_post="true" reload_on_attachment="true"/>
```

Reply:
As long as attributes are truthy, so we could even have `="1"` like in `<form/>` for the `<chatter/>` attributes. Lazy devs will appreciate it.

This PR is a small but quite nice improvements that we kept not doing for far too long! ;)